bedrock-devnet/devnet/__init__.py

```diff
@@ -55,15 +56,15 @@ def main():
 
     monorepo_dir = os.path.abspath(args.monorepo_dir)
     devnet_dir = pjoin(monorepo_dir, '.devnet')
-    contracts_bedrock_dir = pjoin(monorepo_dir, 'packages', 'tokamak', 'contracts-bedrock')
+    contracts_bedrock_dir = pjoin(monorepo_dir, 'packages', 'contracts-bedrock') if args.legacy else pjoin(monorepo_dir, 'packages', 'tokamak', 'contracts-bedrock')
```

Comment:
I am not sure if we can use "if not" instead of " if args.legacy else"

Reply:
In fact, I didn't know Python grammer well. I should study more, I'm sorry. Let me update it! 🙇🏻‍♂️

Reply:
Oh, i am sorry. I dont know much about Python, Please keep it

Reply:
In fact, I want to do like your opinion. 😄
I tried !, but it is not working. 😊 kkk

I learned the correct grammer is the 'not' thanks to you!
I will test and update it tomorrow! 🙇🏻‍♂️

Reply:
you can use

```suggestion
    if args.legacy:
      contracts_bedrock_dir = pjoin(monorepo_dir, 'packages', 'contracts-bedrock')
    else:
      contracts_bedrock_dir = pjoin(monorepo_dir, 'packages', 'tokamak', 'contracts-bedrock')
```
```

Reply:
With Python, I like shorter source code more. It is personal opinion. ^^

Reply:
I like shorter too. Please forgive me @0x6e616d,,,🥰